Tasks/Churo/task_3/Task_3.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+while True:
+    line_length = (input('input line length would you want, but more then 35 elements : \n'))
+    if not re.findall(r'^\d+$', line_length):
+        continue
+    line_length = int(line_length)
+
+    if line_length <= 35:
+        print(f"Incorrect input! The value 'line_length' does not match the condition. Please, try again \n")
+        continue
+    else:
+        break
+
+with open('text.txt', 'r+') as source_file:
+    content = source_file.readlines()
```

Comment:
reading the whole file in memory is not the best approach since it can be very big